season/string_manipulation.md

```diff
@@ -1,6 +1,6 @@
 # String Manipulation
 
-## Allen Iverson
+## Allen Allen_Iverson_Practice
 
```

Comment:
Why did you change the name here? ⛹️

Reply:
i have no idea how that happened. Sorry.... :(